File: VB/NavigationDemo.Wpf/ViewModels/NavigationPane/InfoViewModel.vb
Imports DevExpress.Mvvm

Namespace NavigationDemo
    Public Class InfoViewModel
        Implements ISupportParameter

        Public Overridable Property Parameter() As Object

        #Region "ISupportParameter"
        Private Property ISupportParameter_Parameter() As Object Implements ISupportParameter.Parameter
            Get
                Return Parameter
            End Get
            Set(ByVal value As Object)
                Parameter = value
            End Set
        End Property
        #End Region
    End Class
End Namespace

File: CS/NavigationDemo.Wpf/ViewModels/NavigationPane/InfoViewModel.cs
using DevExpress.Mvvm;

namespace NavigationDemo {
    public class InfoViewModel : ISupportParameter {
        public virtual object Parameter { get; set; }

        #region ISupportParameter
        object ISupportParameter.Parameter {
            get { return Parameter; }
            set { Parameter = value; }
        }
        #endregion
    }
}
